jdbc/src/test/java/org/apache/metamodel/jdbc/integrationtests/DB2Test.java

```diff
@@ -26,6 +26,8 @@
 import org.apache.metamodel.query.Query;
 import org.apache.metamodel.schema.Schema;
 import org.apache.metamodel.schema.Table;
+import org.junit.Test;
+import static org.junit.Assert.*;
```

Comment:
Static imports are usually put at the top of the import list. (this comment applies to all java files where you added static imports). If your using an IDE, check if it contains an "Organize Imports" option. If so, please apply it to the files you changed. It should move the static imports to the top (and maybe reorganize some other imports too).

Reply:
My IDE (IntelliJ Ultimate 2020) seems to automatically move the static import to the bottom, so I just manually moved the static ones to the top, for consistency with this codebase. The "organize import" also pulled in some new imports, so let me know if I should go back and remove them.